frontend/express/public/core/user-management/templates/drawer.html

```diff
@@ -17,43 +17,44 @@
                 <div class="user-management-drawer-content__label">
-                  <div class="text-small text-heading">Full Name</div>
+                  <div class="text-small text-heading">{{ i18n('management-users.full-name') }}</div>
                   <div class="user-management-drawer-content__help-text"></div>
                 </div>
                 <div class="user-management-drawer-content__input">
                   <validation-provider rules="required" v-slot="v">
-                    <el-input v-model="drawerScope.editedObject.full_name" placeholder="Enter Full Name"></el-input>
+                    <el-input v-model="drawerScope.editedObject.full_name" :placeholder="i18n('management-users.enter-full-name')"></el-input>
                   </validation-provider>
                 </div>
               </div>
-              <div class="user-management-drawer-content__inputs-area__input-wrapper">
+              <div class="user-management-drawer-content__input-wrapper">
                 <div class="user-management-drawer-content__label">
-                  <div class="text-small text-heading">Username</div>
+                  <div class="text-small text-heading">{{ i18n('management-users.username') }}</div>
                   <div class="user-management-drawer-content__help-text"></div>
                 </div>
                 <div class="user-management-drawer-content__input">
                   <validation-provider rules="required" v-slot="v">
-                    <el-input v-model="drawerScope.editedObject.username" placeholder="Enter User Name"></el-input>
+                    <el-input v-model="drawerScope.editedObject.username" :placeholder="i18n('management-users.enter-username')"></el-input>
                   </validation-provider>
                 </div>
               </div>
-              <div class="user-management-drawer-content__inputs-area__input-wrapper">
+              <div class="user-management-drawer-content__input-wrapper">
                 <div class="user-management-drawer-content__label">
-                  <div class="text-small text-heading">E-mail</div>
+                  <div class="text-small text-heading">{{ i18n('management-users.email') }}</div>
                   <div class="user-management-drawer-content__help-text"></div>
                 </div>
                 <div class="user-management-drawer-content__input">
                   <validation-provider rules="required|email" v-slot="v">
-                    <el-input v-model="drawerScope.editedObject.email" placeholder="Enter E-mail"></el-input>
+                    <el-input v-model="drawerScope.editedObject.email" :placeholder="i18n('management-users.enter-email')"></el-input>
                   </validation-provider>
                 </div>
               </div>
+              <component v-if="groupsInput.length > 0" :defaultGroup="group" @group-change="onGroupChange" :is="groupsInput[0].component"></component>
```

Comment:
i believe you can use dynamic tabs here too? no?

Reply:
yes and no. I explained it on DM.